src/components/file-preview.less

```diff
@@ -0,0 +1,68 @@
+.FilePreview {
+  display: grid;
+  gap: 10px;
+  grid-template-rows: auto 1fr;
+  grid-template-columns: 1fr 1fr;
+}
+
+.FilePreview-headerArea {
+  grid-row: 1;
+  grid-column-start: 1;
+  grid-column-end: 3;
+}
+
+.FilePreview-previewArea {
+  grid-row: 2;
+  grid-column: 1;
+  padding: 1em;
+  height: 24em;
```

Comment:
I feel like we're getting pretty inconsistent with the units we use for dimensions and positioning. Do you think it's absolutely needed in this case to work with `em` (I understand the arguments for this, generally), or could we stick to `px` here? 🤔

Reply:
Sure, no problem, switched most of them to `px` (except for font related rules)